src/test/common/wiremock.js

```diff
@@ -0,0 +1,26 @@
+const { postJsonData, performDelete } = require('./request')
+
+const WIREMOCK_MAPPING_URL = process.env.WIREMOCK_URL ? `${process.env.WIREMOCK_URL}/__admin/mappings` : 'http://localhost:8090/__admin/mappings'
+
+const SUCCESSFUL_CLAIMS_MAPPING = `{
```

Comment:
You could maybe use `JSON.stringify()` here - a template string feels a bit odd, but maybe its OK?